pyproject.toml

```diff
@@ -18,7 +18,7 @@ beautifulsoup4 = "~=4.9"
 colorama = { version = "~=0.4.3", markers = "sys_platform == 'win32'" }
 coloredlogs = "~=14.0"
 deepdiff = "~=4.0"
-"discord.py" = "~=1.6.0"
+"discord.py" = "~=1.7.3"
```

Comment:
Why does this PR increase the discord.py version?

Reply:
Stage channels were introduced in 1.7 ([reference](https://discordpy.readthedocs.io/en/latest/api.html#discord.ChannelType.stage_voice)).

Reply:
Fair, enough my bad, I seem to have overlooked that part in the message.